src/js_native_api_v8.cc

```diff
@@ -2569,6 +2569,38 @@ napi_status napi_create_arraybuffer(napi_env env,
   return GET_RETURN_STATUS(env);
 }
 
+napi_status napi_is_detached_arraybuffer(napi_env env,
+                                         napi_value value,
+                                         napi_value* result) {
+  NAPI_PREAMBLE(env);
```

Comment:
I don't think we need `NAPI_PREAMBLE()` here, because we don't have any `Maybe`s.